reg_tests/rt.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+export MAILTO=
+
+# Directory to download UFS_UTILS to and run the regression tests
+export WORK_DIR=
+
+export PROJECT_CODE=
+export QUEUE=
+
```

Comment:
@kgerheiser I am getting an error at line 11.  It is looking for sorc/machine-setup.sh in my home directory.  Cron jobs on Dell default to a user's home directory.  This would work if I checked out UFS_UTILS in my home directory.  But we don't get much space in our home dir.  I think you need to move this logic to after the clone.  We cant _run_  anything on WCOSS-prod, but we can clone and build on prod.